images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -193,12 +193,16 @@ $markdown += New-MDList -Style Unordered -Lines @(
 )
 
 $markdown += New-MDHeader "Browsers and Drivers" -Level 3
-$markdown += New-MDList -Style Unordered -Lines @(
+$browsersAndDriversList = @(
     (Get-ChromeVersion),
     (Get-ChromeDriverVersion),
     (Get-FirefoxVersion),
     (Get-GeckodriverVersion)
 )
+if (Test-IsUbuntu20) {
+    $browsersAndDriversList += @(Get-ChromiumVersion)
```

Comment:
I think we need to reorder it a bit:
1) ChromiumVersion
2) ChromeVersion
3) ChromeDriverVersion
4) FirefoxVersion
5) GeckodriverVersion

The reason for this order is that Chrome and Chromium are almost identical and use the same ChromeDriver.

Reply:
Done